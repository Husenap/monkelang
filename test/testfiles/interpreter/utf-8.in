monke say "안녕하세요"
monke say "你好"
monke say "こんにちは"
monke say "مرحبا"
monke say "🙈🙉🙊🐵🐒😍🦧"
