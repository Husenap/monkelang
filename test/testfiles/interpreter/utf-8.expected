안녕하세요
你好
こんにちは
مرحبا
🙈🙉🙊🐵🐒😍🦧
